### PR TITLE
refactor: [ANDROAPP-4431] remove picture view and viewmodel

### PR DESCRIPTION
**Pull request by mmmateos**
## Description
Please include a summary of the change and include the related jira issue if it exists.

[ jira issue ](https://jira.dhis2.org/browse/

## Solution description
If this PR is a fix include a brief description on how the issue is solved.
## Covered unit test cases
Describe the tests that you ran to verify your changes.
## Where did you test this issue?
- [ ] Smartphone Emulator
- [ ] Tablet Emulator
- [ ] Smartphone
- [ ] Tablet
## Which Android versions did you test this issue?
- [ ] 4.4
- [ ] 5.X - 6.X
- [ ] 7.X
- [ ] 8.X
- [ ] 9.X - 10.X
- [ ] Other
## Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code

### CODE REVIEW

File: form/src/main/res/layout/form_picture.xml [hunk 1]
```diff
@@ -6,22 +6,18 @@
...
-
-        <variable
-            name="description"
-            type="java.lang.String" />
+            name="item"
+            type="org.dhis2.form.model.FieldUiModel" />
 
     </data>
 
     <androidx.constraintlayout.widget.ConstraintLayout
```
Comment (andresmr) [inline form/src/main/res/layout/form_picture.xml:17]: could we move this layout to form module?

File: form/src/main/java/org/dhis2/form/ui/FormViewModel.kt [hunk 2]
```diff
@@ -173,7 +173,10 @@ class FormViewModel(
     ): Throwable? {
         return fieldValue?.let { value ->
             var error =
-                when (val result = valueType?.validator?.validate(value)) {
+                when (
+                    val result = valueType?.takeIf { it != ValueType.IMAGE }
```
Comment (andresmr) [inline form/src/main/java/org/dhis2/form/ui/FormViewModel.kt:177]: ValueType.IMAGE has the following validator:
` IMAGE(String::class.java, UidValidator);`
Is this class: UidValidator.kt. Why can not be validated?

File: form/src/main/res/layout/form_picture.xml (hunk 1 above)
Comment (mmmateos) [reply form/src/main/res/layout/form_picture.xml:17]: I think is already in the form module :smile:

File: form/src/main/java/org/dhis2/form/ui/FormViewModel.kt (hunk 2 above)
Comment (mmmateos) [reply form/src/main/java/org/dhis2/form/ui/FormViewModel.kt:177]: The problem is when reaching this validator the value of image field is not a UID, is the path of the file that has to be stored. Maybe we could discuss if this validator could be skipped or we should make something else.

**Pull request merged**